src/content/platform-overview/cluster-management/product/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Our product
+description: Overview of the Giant Swarm cluster management product, the different generations and cloud provider implementations.
```

Comment:
```suggestion
description: Overview of the Giant Swarm Platform, the different generations and cloud provider implementations.
```
I do think we need to talk about the platform since it is not just cluster mgmt, we want to adapt the documentation to look at the coming future. The idea is to serve a dev platform to the customers and we can start building over it

Reply:
I have seen this is under cluster mgmt but even that I would not call it product since the product IMO it is the platform. Obviously, it has parts and layers, we could start showing a graphic in the `platform overview` and then here talk about the specifics of the cluster mgmt. But changing the naming from `product` to flavour.